genisys/modules/preseed.py

```diff
@@ -4,51 +4,58 @@
 from genisys.modules.base import Module
 from genisys.config_parser import YAMLParser
 import sys
-FILENAME = "preseed.cfg"
 
+FILENAME = "preseed.cfg"
+CERTIFICATE_STORE_PATH = Path("/etc/genisys/ssl")
```

Comment:
This is already defined in `genisys.server.tls` we don't want to replicate that definition incase we need to change it later we don't have to start hunting down multiple places it's used. You can use this value in your code below by first `from genisys.server.tls import CERTIFICATE_STORE_PATH`